### PR TITLE
SEO-2900 XIsIO UTM URL Change

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @ChristoIssac ,

Please review the changes for UTM URL Changes.

Regards,
Yvone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,15 +52,15 @@ Syncfusion Essential XlsIO supports .xls, .xlsx, .xlsm, .xlsb, .xltm, .xltx, .xl
...
+This is a commercial product and requires a paid license for possession or use. Syncfusion�s licensed software, including this component, is subject to the terms and conditions of [Syncfusion's EULA](https://www.syncfusion.com/eula/es/?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples). You can purchase a licnense [here](https://www.syncfusion.com/sales/products?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples) or start a free 30-day trial [here](https://www.syncfusion.com/account/manage-trials/start-trials?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples).
```
Comment (Girija-Rajendran) [inline README.md:55]: check the special characters
image

File: README.md [hunk 2]
```diff
@@ -52,15 +52,15 @@ Syncfusion Essential XlsIO supports .xls, .xlsx, .xlsm, .xlsb, .xltm, .xltx, .xl
...
+This is a commercial product and requires a paid license for possession or use. Syncfusions licensed software, including this component, is subject to the terms and conditions of [Syncfusion's EULA](https://www.syncfusion.com/eula/es/?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples). You can purchase a licnense [here](https://www.syncfusion.com/sales/products?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples) or start a free 30-day trial [here](https://www.syncfusion.com/account/manage-trials/start-trials?utm_source=github&utm_medium=listing&utm_campaign=github-xlsio-examples).
```
Comment (Girija-Rajendran) [inline README.md:55]: Syncfusion's licensed..........

File: README.md [hunk 3]
```diff
@@ -52,15 +52,15 @@ Syncfusion Essential XlsIO supports .xls, .xlsx, .xlsm, .xlsb, .xltm, .xltx, .xl
...
...s?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), and [JavaScript](https://www.syncfusion.com/javascript-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples)), and desktop development ([Windows Forms](https://www.syncfusion.com/winforms-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [WPF](https://www.syncfusion.com/wpf-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [WinUI(Preview)](https://www.syncfusion.com/winui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples), [Flutter](https://www.syncfusion.com/flutter-widgets?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples) and [UWP](https://www.syncfusion.com/uwp-ui-controls?utm_source=github&utm_medium=github&utm_campaign=github-xlsio-examples)). We provide ready-to-deploy enterprise software for dashboards, reports, data integration, and big data processing. Many customers have saved millions in licensing fees by deploying our software.
```
Comment (Girija-Rajendran) [inline README.md:60]: remove preview for WinUI
image

Should add MAUI platform to this list

File: README.md (hunk 2 above)
Comment (Yvone-Atieno) [reply README.md:55]: changed

File: README.md (hunk 3 above)
Comment (Yvone-Atieno) [reply README.md:60]: added maui platform and removed preview

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**